cmd/mesh/manifest.go

```diff
@@ -179,6 +191,10 @@ func ignoreError(stderr string) bool {
 	return trimmedStdErr == ""
 }
 
+func isDisabledComponent(out *manifest.ComponentApplyOutput) bool {
```

Comment:
This is not the most robust way to check. It's better to use the input manifest set, or even better ICPS.

Reply:
Agree that we should check ICPS for enable spec.

FYI, this function also checks if any action (e.g., prune) has been done to this disabled component, based on the out.std.Stdout. What we can do here is to categorize the kubectl execution results.

Reply:
rather than categorizing the output, how about just binding the original input?

Reply:
Updated, please take a look.